<launch>
  <arg name="veh" default="catvehicle"/>

  <group ns="$(arg veh)">

    <node name="overtaking_maneuver_node" pkg="overtaking_maneuver" type="overtaking_maneuver_node" respawn="false" output="screen">
      <!--remap from="overtaking_path" to="control_path"/-->
      <remap from="overtaking_path" to="route_plan"/>
    </node>

    <node name="button_simulator_node" pkg="overtaking_maneuver" type="button_simulator" respawn="false" output="screen">
    </node>

    <node name="overtaking_state_machine_node" pkg="overtaking_maneuver" type="overtaking_state_machine" respawn="false" output="screen">
    </node>

  </group>

</launch>
